GameData/RSF/RSF_Compatibility/RSF_ProceduralParts.cfg

```diff
@@ -1,21 +1,21 @@
 //
 // Changes Procedural part diameters to fit RSF start
 //
-@PART[proceduralStackDecoupler]:NEEDS[LRTRTechTree]:AFTER[zLRTR] // Changes stack decoupler to be more start friendly
+@PART[proceduralStackDecoupler]:AFTER[zLRTR]:NEEDS[LRTRTechTree] // Changes stack decoupler to be more start friendly
 {
 	@MODULE[ProceduralPart]
 	{
 	
-		@diameterMin = 0.35
+		@diameterMin = 0.375
 	}
 }
 
-@PART[proceduralTankLiquid|proceduralTankRealFuels]:NEEDS[LRTRTechTree]:AFTER[zLRTR]
+@PART[proceduralTankLiquid|proceduralTankRealFuels]:AFTER[zLRTR]:NEEDS[LRTRTechTree]
 {
         
     @MODULE[ProceduralPart]
     {
-        @diameterMin = 0.1
+        @diameterMin = 0.375
```

Comment:
I like to make really tiny pill tanks. We can reintroduce the min diameter in the upgrade path though.